dataplane/saiserver/hostif.go

```diff
@@ -235,6 +249,8 @@ func (hostif *hostif) CreateHostif(ctx context.Context, req *saipb.CreateHostifR
 
 func (hostif *hostif) createRemoteHostif(ctx context.Context, req *saipb.CreateHostifRequest) (*saipb.CreateHostifResponse, error) {
 	id := hostif.mgr.NextID()
+	hostif.remoteMu.Lock()
```

Comment:
I think remoteMu is used to guard remoteHostifs[], should we move it closer to where remoteHostifs is acutally used?

Reply:
done